docs/_documentation/configuration.md

```diff
@@ -77,6 +77,30 @@ In addition to `inline` strategy there is also the `async` option. This will add
 the async property to the injected script tag, which will make the loading
 behavior of the registration script async.
 
+By default the ServiceWorker is registered on the same scope as the Ember
+application's `rootURL`. If you need to set a scope that is not the same as the
+`rootURL`, you an use the `serviceWorkerScope` config option to override the
+default.
+
+```js
+var EmberApp = require('ember-cli/lib/broccoli/ember-app');
+
+module.exports = function(defaults) {
+  var app = new EmberApp(defaults, {
+    'ember-service-worker': {
+      serviceWorkerScope: '/v2/api/'
+    }
+  });
+
+  return app.toTree();
+};
+```
+
+You can read more about ServiceWorker scope on [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API/Using_Service_Workers#Enter_service_workers).
```

Comment:
Can we also add a caution regarding that the scope can only _narrow_ the scope of the service worker?
For example:

> Since a service worker can't have a scope broader than its own location, only use the scope option when you need a scope that is narrower than the default.

from [MDN](https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerContainer/register)

Reply:
You did this in https://github.com/meirish/ember-service-worker/pull/1 🙌